### DISCUSSION TITLE
can I nest hybrid(+) folders?

### DISCUSSION THREAD

**Discussion by tomer-yechiel** [kiliman/remix-flat-routes#91] [Q&A] [answered]
I am trying to make the route /dashboard/availability(file routes/dashboard+/availability+/index.tsx)
to have the layout in the file routes/dashboard+/availability+/layout.tsx
<img width="284" alt="image" src="https://github.com/kiliman/remix-flat-routes/assets/32360493/b89f862d-b650-439b-a098-d5fe10e2ff9e">

**Comment by kiliman** [comment, marked as answer]
With *hybrid* routes, the layout route should be named `_layout.tsx`.

`routes/dashboard+/availability+/_layout.tsx`

This is the layout route for all child routes `routes/dashboard+/availability+/*`

See https://github.com/kiliman/remix-flat-routes#nested-folders-with-flat-files-convention--new-in-v051

**Reply by tomer-yechiel** [reply]
Thanks!
